### DISCUSSION TITLE
Consent support in gatsby-plugin-google-gtag

### DISCUSSION THREAD

**Discussion by imreFitos** [gatsbyjs/gatsby#35928] [Ideas / Feature Requests]
I'd like the gatsby-plugin-google-gtag plugin to support consent settings like Google describes here: https://support.google.com/analytics/answer/9976101?hl=en

**Comment by imreFitos** [comment]
so here is the PR to implement this: https://github.com/gatsbyjs/gatsby/pull/35929

**Comment by george-thomas-hill** [comment]
Having this would be great!
